deploy/charts/bottlerocket-update-operator/values.yaml

```diff
@@ -111,3 +111,7 @@ logging:
     tracing_filter: "info"
   apiserver:
     tracing_filter: "info"
+
+prometheus:
+  serviceMonitor:
+    enabled: false
```

Comment:
Do you mind fixing the missing newline at EOF?

Reply:
Added missing newline